lib/src/clojure_lsp/queries.clj

```diff
@@ -79,20 +79,26 @@
        (= (:from usage) (:to usage))))
 
 (defn find-local-usages-under-form
-  [analysis filename line column end-line end-column]
-  (let [local-analysis (get analysis filename)]
+  [analysis filename row col end-row end-col]
```

Comment:
These changes aren't crucial to the feature... I was just trying to understand how this code works and decided to clarify it a little.